sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentModelAdministrationClient.cs

```diff
@@ -114,8 +114,12 @@ public DocumentModelAdministrationClient(Uri endpoint, TokenCredential credentia
         /// <summary>
         /// Build a custom document analysis model from a collection of documents in a blob storage container.
         /// </summary>
-        /// <param name="trainingFilesUri">An externally accessible Azure storage blob container Uri pointing to the container that has your training files.
-        /// For more information see <see href="https://docs.microsoft.com/azure/applied-ai-services/form-recognizer/build-training-data-set">here</see>.</param>
+        /// <param name="trainingFilesUri">
+        /// An externally accessible Azure storage blob container URI pointing to the container that has your training files.
+        /// Note that a container URI without SAS is accepted only when the container is public or has a managed identity
+        /// configured.
```

Comment:
I know in Python we linked to the managed identity docs. would that make sense here?

Reply:
I preferred to keep it simple and avoided adding more references.